web/assets/js/app/Browser/component/controller/NodeController.js

```diff
@@ -3,16 +3,18 @@ define([
 ], function(angular) {
     'use strict';
 
-    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $notification, $treeFactory, $timeout, $q) {
+    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $treeFactory, $timeout, $q, $success, $error, $errorFromResponse) {
```

Comment:
What's the rationale about spitting the notification service in two? It makes signatures and code harder to follow, IMO.